CONTRIBUTING.md

```diff
@@ -24,6 +24,23 @@ Check out our [README.md] for more details.
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
+## Our Artifacts
+
+In addition to our [Kanban board] cards, we produce the following artifacts.
+
+```mermaid
```

Comment:
Does this render as a mermaid diagram for you?
I am looking at this on GitHub mobile app, where it is shown as syntax highlighted code rather. Might be a shortcoming if the app though.

Reply:
Yes, it renders.